### PR TITLE
Repairing robotic organs is now of type /decl/surgery_step/internal because they can be inside meat

### LINKED ISSUE #9022: Augments can't be repaired
It appears that augments can't be repaired. After opening a surgical site with a scalpel, neither advanced brute pack nor nanopaste works to repair damage done to the augments.

### LINKED ISSUE #9203: Retractable Combitool not fixable. Neither nanopaste nor ATK can fix it.
Checklist before you submit an issue! Feel free to partially or fully delete this in your eventual report.
* Do a quick key word search of the git for duplicate reports. If you find any, simply post a reply onto that issue instead!
* Please be specific in your description of the issue. Explain the desired result (what should be happening) and the actual result (what is happening). Simply stating that [something happens] is not very helpful at describing what's wrong.
* Have you tried reproducing the issue? If yes, steps to reproduce it would help immensely!
* All additional details help. Such as round ID, an admin you talked to if the issue required immediate fixing, etcetera, etcetera.

### PR DESCRIPTION
**Pull request by mikomyazaki**
Fixes #9022
Fixes #9203

/decl/surgery_step/robotics is for things involving robotic *external* organs, and things within them.
/decl/surgery_step/internal is for things involving meat OR robotic *external* organs, and things within them.

### CODE REVIEW

**Comment by mikomyazaki** [comment]
!bugfix

**Comment by mikomyazaki** [comment]
!review

File: code/modules/surgery/organs_internal.dm [hunk 1]
```diff
@@ -100,6 +100,78 @@
...
+
+/decl/surgery_step/internal/fix_organ_robotic/fail_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+	if(!ishuman(target))
+		return
+	var/obj/item/organ/external/affected = target.get_organ(target_zone)
+
+	user.visible_message(SPAN_WARNING("[user]'s hand slips, gumming up the mechanisms inside of [target]'s [affected.name] with \the [tool]!"), \
+		SPAN_WARNING("Your hand slips, gumming up the mechanisms inside of [target]'s [affected.name] with \the [tool]!"))
+
+	target.adjustToxLoss(5)
```
Comment (NonQueueingMatt) [inline code/modules/surgery/organs_internal.dm:167]: shouldn't this damage the organ instead of just adding toxloss?

Comment (mikomyazaki) [reply code/modules/surgery/organs_internal.dm:167]: It does damage the organ, see line 172.

Comment (NonQueueingMatt) [reply code/modules/surgery/organs_internal.dm:167]: Bruh

**Pull request merged**